packages/editor/src/client/atoms/currencyInput.tsx

```diff
@@ -23,28 +22,27 @@ export function CurrencyInput({currency, centAmount, disabled, onChange}: Curren
       .replace(',', '.')
       .replace(/^\./, '')
 
-  const {t} = useTranslation()
-
   return (
     <div>
-      <InputGroup>
-        <InputGroup.Addon>{currency}</InputGroup.Addon>
-        <Input
-          value={amount as string}
-          disabled={disabled}
-          onChange={amount => {
-            amount = toFloat(amount)
-            setMessage(!amount)
-            setAmount(amount)
-          }}
-          onBlur={() => {
-            if (amount) {
-              onChange(parseFloat(amount as string) * 100)
-            }
-          }}
-        />
-      </InputGroup>
-      {message ? <Message type="error">{t('memberPlanList.errorMessage')}</Message> : ''}
+      <Form.Group>
+        <InputGroup inside>
+          <Form.Control
+            value={amount as string}
+            name="currency"
+            disabled={disabled}
+            onChange={(amount: string) => {
+              amount = toFloat(amount)
+              setAmount(amount)
+            }}
+            onBlur={() => {
+              if (amount) {
+                onChange(parseFloat(amount as string) * 100)
+              }
+            }}
+          />
+          <InputGroup.Addon>{currency}</InputGroup.Addon>
+        </InputGroup>
+      </Form.Group>
     </div>
```

Comment:
The error message is removed here?

Reply:
Same as for Karl's comment, the error message now comes from validation